src/main/resources/openapi/transcriptions.yaml

```diff
@@ -60,8 +60,43 @@ paths:
                         $ref: '#/components/schemas/RequestTranscription400ErrorCode'
         '401':
           description: Unauthorised Error
+          content:
```

Comment:
Don't think the 401 and 404 errors need expanding here.
They don't look like the standard problem error responses, the body is just empty.

401 is when the Azure access token has expired for example.
404 looks to be similar, so not the one you have added to the schema https://github.com/hmcts/darts-api/blob/master/src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java#L77

Reply:
Removed.